tsdb/wlog/checkpoint.go

```diff
@@ -81,7 +81,7 @@ func DeleteCheckpoints(dir string, maxIndex int) error {
 	return errs.Err()
 }
 
-const checkpointPrefix = "checkpoint."
+const CheckpointPrefix = "checkpoint."
```

Comment:
Since this is exported now, could you add a doc comment what it is for?

Reply:
OK. Done.